packages/flutter_tools/lib/src/base/file_system.dart

```diff
@@ -63,7 +63,20 @@ class FileSystemUtils {
   ///
   /// On Windows it replaces all '\' with '\\'. On other platforms, it returns the
   /// path unchanged.
-  String escapePath(String path) => _platform.isWindows ? path.replaceAll(r'\', r'\\') : path;
+  String escapePath(String path) {
```

Comment:
This can be simplified:

```dart
String escapePath(String path) {
  if (_platform.isWindows) {
    path = path.replaceAll(r'\', r'\\');
    if (path.startsWith(RegExp('[a-z]:')) {
      // ensure that the drive letter is upper case see
      // https://youtrack.jetbrains.com/issue/IDEA-329756/Importing-symlinked-Gradle-included-build-fails#focus=Comments-27-11721320.0-0
      path = path[0].toUpperCase() + path.substring(1);
    }
  }
  return path;
}
```